### ISSUE TITLE
Feature request - Using selectNodes and selectEdges in a row (network)

### ISSUE THREAD

**Issue by b-pos465** [almende/vis#475]
Hey ;)

if I want to use selectNodes and selectEdges in a row the _unselectAll(true) method will be called both times. The result is that one of both changes will be deleted. So i added a new boolean param to both methods, and wrote a new one which combines both:

>exports.selectNodesAndEdges = function(selectionN, selectionE){
>  
>	this.selectNodes(selectionN, false, false);
>	this.selectEdges(selectionE, false);
>	
>  }

> exports.selectNodes = function(selection, highlightEdges, doRedraw) {
>...
>if(doRedraw){
>	this.redraw();
>}

>exports.selectEdges = function(selection, doUnselect) {
>...
>if(doUnselect){
>		this._unselectAll(true);
>	}

What do u think about it? 

Best regards,
Stefan

**Comment by AlexDM0** [comment]
Hi,

Over the last year a lot of feature requests have been made. We have just introduced our new website which has a list of the requested features. We have placed this request on that list. 

The list can be found here:
http://visjs.org/featureRequests.html

An explaination of the new system can be found here:
http://visjs.org/blog.html#New\ website\ for\ vis\.js\!

I would like to stress that this does not mean we abandon this request. Discussion here will continue if needed on this feature but we will close it to keep our Github issue page more of a bug-todo list.

Future feature requests will still be made here and then added to the website by us.

Regards,

Alex
